ui/helpers/sort.js

```diff
@@ -24,3 +23,30 @@ export const sortAlphaNum = (a, b) => {
     return rv;
   }
 };
+
+// This is a helper function for sorting an array of hashes:
+// arr.sort(sortHashes(key, dataType, desc))
+export const sortHashes = (key, dataType, desc) => {
```

Comment:
I think it'd be clearer to name this differently. Something like `determineSortByType` or `getSortOrder`.

Reply:
Got the naming standard from the function above :)